src/components/quicklinks.tsx

```diff
@@ -0,0 +1,104 @@
+import { KeyboardEvent, useEffect, useState } from "react";
+import { QUICK_LINKS } from "../constants/appConstants";
+
+import ArrowTop from "../assets/arrow_top.svg";
+import ArrowTopWhite from "../assets/arrow_top_white.svg";
+import CustomImage from "./common/customimage";
+import CustomTab from "./common/customtab";
+import useScreenSize from "../hooks/useScreenSize";
+import CustomButton from "./Button";
+
+const QuickLinks = () => {
+  const [tabValue, setTabValue] = useState<number>(0);
+  const [quickLinks, setQuickLinks] = useState([]);
+  const [allQuickLinks, setAllQuickLinks] = useState([]);
+  const [hoveredIndex, setHoveredIndex] = useState<number>(null);
+  const [showSeeMoreBtn, setShowSeeMoreBtn] = useState<boolean>(false);
+  const { width } = useScreenSize();
+
+  const onSeeMoreClick = () => {
+    setQuickLinks(allQuickLinks);
+    setShowSeeMoreBtn(false);
+  }
+
+  const getQuickLinks = () => {
+    let links = QUICK_LINKS.find(l => l.id === tabValue).links;
+    setQuickLinks(links);
+    setAllQuickLinks(links);
+  }
+
+  useEffect(() => {
+    getQuickLinks();
+  }, [tabValue]);
+
+  useEffect(() => {
+    if (width < 700 && allQuickLinks.length > 0 && !(allQuickLinks.length < 10)) {
+      let _quickLinks = [...allQuickLinks];
+      _quickLinks.splice(10);
+      setQuickLinks(_quickLinks);
+      setShowSeeMoreBtn(true);
+    } else {
+      setShowSeeMoreBtn(false);
+      setQuickLinks(allQuickLinks);
+    }
+  }, [width, allQuickLinks]);
+
+  return (
+    <div className="p-4 sm:px-8 sm:py-6 bg-[#22262C] quick-links absolute top-[56px] sm:left-[25vw] lg:left-[21vw] right-0">
+      <div className="flex flex-wrap justify-between items-center pb-2">
+        <span className="text-[#FAF9F6] pb-2 font-bold">Quick Links</span>
```

Comment:
Keep constant string like `Quick Links` in `appconstants.tsx` so that it is easy to edit or change in future.